apps/web-twig-demo/docker/docker-compose.override.yml

```diff
@@ -23,13 +23,14 @@ services:
 
   web-twig-demo-encore:
     container_name: web-twig-demo-encore
-    command: 'sh -c "yarn && yarn watch"'
+    command: 'sh -c "sleep 10 && yarn && yarn watch"'
```

Comment:
This is not working correctly because Yarn tries to install the demo package from the internet anyway.

Reply:
Removed. Fix by publishing the package.